pom.xml

```
@@ -1172,6 +1172,7 @@
                 <module>samples/client/petstore/rust</module>
                 <module>samples/client/petstore/rust/reqwest/petstore</module>
```

nit: Hyper directory is missing as well
```suggestion
                <module>samples/client/petstore/rust/hyper/petstore</module>
                <module>samples/client/petstore/rust/reqwest/petstore</module>
```

---

don't recall why it's omitted. agreed it should be included if the test passes.